help/assets/css/styles.scss

```diff
@@ -0,0 +1,354 @@
+---
```

Comment:
NAB I don't think these lines `---` are necessary here.

Reply:
Ah these lines are needed for Jekyll https://jekyllrb.com/docs/step-by-step/07-assets/#sass otherwise the styles are broken (I tried it just for curiosity hehe):

<img width="1783" alt="Screen Shot 2022-07-01 at 14 30 29" src="https://user-images.githubusercontent.com/6829422/176958339-2620e9b1-714f-4db2-9006-08d9126e0cd2.png">